### DISCUSSION TITLE
TAVMedia是不是包含PAG及其企业版的功能, 类似对标Galacean Effects?

### DISCUSSION THREAD

**Discussion by luciddream-tsin** [Tencent/libpag#2176] [Q&A] [answered]
TAVMedia是不是包含PAG及其企业版的功能, 类似对标Galacean Effects?

TAVMedia是一个编辑器吗， 是否可以在一些简单需求上替代AE?

付费用户并且是否提供源码?

**Comment by kevingpqi123** [comment, marked as answer]
TAVMedia 主要针对的是服务端渲染，针对视频后编辑，向视频中添加特效，典型的应用场景是视频广告和游戏战报
由于目前人力有限，而这部分对接需要花费较大的人力，目前已经停止外部对接，感谢您的关注
